docs/spark/spark-writes.md

```diff
@@ -311,7 +311,9 @@ distribution & sort order to Spark.
 {{< /hint >}}
 
 {{< hint info >}}
-Both global sort (`orderBy`/`sort`) and local sort (`sortWithinPartitions`) work for the requirement.
+With SQL, you can use `ORDER BY` to achieve global sorting or `SORT BY` to achieve local sorting.
```

Comment:
Could you please add the `ORDER BY` and `SORT BY` links from apache spark pages ?

Reply:
Of course, I updated the commit, please help to review it again, thanks!